### PR TITLE
Prior Investigation Date Fix

### PR DESCRIPTION
**Pull request by jgoodell-gsa**
* **Add** trim to model
* **Add** validation for white space
* **Add** code to insert DBNull if not a valid date
* [Release script](https://drive.google.com/open?id=10YPuK5rERz9k3xS33aFm901rJ7JB8V33hJrcwlc37eQ)

### CODE REVIEW

**Comment by jgoodell-gsa** [comment]
Initial testing successful.

File: ProcessCIW/Models/CIW.cs [hunk 1]
```diff
@@ -109,7 +110,11 @@ public string PersonalEmailAddress
 
         public string PositionJobTitle { get; set; }
         public string PriorInvestigation { get; set; }
-        public string ApproximiateInvestigationDate { get; set; }
+        public string ApproximiateInvestigationDate
```
Comment (tsaunders-gsa) [inline ProcessCIW/Models/CIW.cs]: Please add comment that this is the prior investigation date.

File: ProcessCIW/Models/CIW.cs [hunk 2]
```diff
@@ -41,6 +41,7 @@ class CIW
         private string _SponsorAlternateEmailAddress2;
         private string _SponsorAlternateEmailAddress3;
         private string _SponsorAlternateEmailAddress4;
+        private string _ApproximiateInvestigationDate;
```
Comment (tsaunders-gsa) [inline ProcessCIW/Models/CIW.cs]: Please add comment that this is the prior investigation date.

**Comment by jgoodell-gsa** [comment]
Comments added

File: ProcessCIW/Models/CIW.cs (hunk 2 above)
Comment (jgoodell-gsa) [reply ProcessCIW/Models/CIW.cs]: Comment added to identify Column name in database that corresponds with _ApproximiateInvestigationDate.

File: ProcessCIW/Models/CIW.cs (hunk 1 above)
Comment (jgoodell-gsa) [reply ProcessCIW/Models/CIW.cs]: Comment added to identify Column name in database that corresponds with ApproximiateInvestigationDate.

**Comment by tsaunders-gsa** [comment]
This has been merged into dev.

**Pull request merged**